interfaces/PA-App/src/app/tabs/popover/popover.component.ts

```diff
@@ -33,32 +32,12 @@ export class PopoverComponent implements OnInit {
     this.popoverController.dismiss();
   }
 
-  logout() {
+  async logout() {
     this.paData.logout();
     this.paData.store('isLoggedIn', false, true);
     this.popoverController.dismiss();
-    this.createToast(this.translate.instant('account.logged-out'));
+    await this.ionStorage.clear();
```

Comment:
Actually; I think it shouldn't be the responsibility of the "component-that-triggers-the-log-out" to clear the storage...

I think `this.paData.logout()` should "clean-up-after-itself", so in there we should clear the local-storage...

Reply:
`window.location.reload();` would be the exception, I think...
As that should be 'connected' the the user-action of pressing the button, in some way...